doc/geany.txt

```diff
@@ -3357,6 +3357,10 @@ Delete to line start            Ctrl-Shift-BackSpace      Deletes from the begin
                                                           current caret position.
 
 Duplicate line or selection     Ctrl-D                    Duplicates the current line or selection.
+							  If the selection spans more than one line,
+							  this duplicates the lines containing selections.
+							  If the selection spans only a single line, then only
+							  the selection is duplicated.
 
```

Comment:
Dunno, sounds more confusing than the "right thing", but anyway as you can tell I don't use it :)

Then a list just like your three dot points then, simple is better :)  The only thing would be that the middle point might say "if the selection is on one line only, only the selection is duplicated"